nixos/modules/rename.nix

```
@@ -20,6 +20,7 @@ in
     (mkAliasOptionModuleMD [ "environment" "checkConfigurationOptions" ] [ "_module" "check" ])
 
     # Completely removed modules
+    (mkRemovedOptionModule [ "boot" "loader" "raspberrypi" ] "The raspberrypi boot loader has been removed.")
```

```suggestion
    (mkRemovedOptionModule [ "boot" "loader" "raspberryPi" ] "The raspberryPi boot loader has been removed.")
```

It might also be nice to tell people what they should be using instead.